### ISSUE TITLE
Allow disabling symbol resolution for some modules

### ISSUE THREAD

**Issue by UnrealKaraulov** [VerySleepy/verysleepy#109]
Need show "DLL+Offset", but I see only "FIRSTFUNCTIONNAME".

For example I have one dll "External.DLL" with one exported function "FIRSTFUNCTIONNAME"

For example I need see in stack something like:
```
External.DLL+0x502C 
External.DLL+0x3062
External.DLL+0x2004
External.DLL+0x600C
External.DLL+0x3062
....
```

But I see only
```
FIRSTFUNCTIONNAME
FIRSTFUNCTIONNAME
FIRSTFUNCTIONNAME
FIRSTFUNCTIONNAME
FIRSTFUNCTIONNAME
FIRSTFUNCTIONNAME
.....
```

**Comment by CyberShadow** [comment]
> Need show "DLL+Offset", but I see only "FIRSTFUNCTIONNAME".

Yeah, I think it's because dbghelp will use exports as symbols if there are no symbols.

I think we need a right-click menu option to disable trying to resolve symbols in the clicked module, and show offsets instead.

> Fatal error: No offsets in stack

I'm not sure what this has to do with the above. Is this a different problem?

**Comment by UnrealKaraulov** [comment]
> I'm not sure what this has to do with the above. Is this a different problem

Okay I do rename this issue

**Comment by UnrealKaraulov** [comment]
> I think we need a right-click menu option to disable trying to resolve symbols in the clicked module, and show offsets instead.

Yes! (Only for "whitelisted" dll's for example)


You can fix it?

**Comment by CyberShadow** [comment]
If you would like to add this feature, I'll try to review the PR and get it in.

**Comment by UnrealKaraulov** [comment]
> If you would like to add this feature, I'll try to review the PR and get it in.

I can't build this tool from build.cmd :(

**Comment by CyberShadow** [comment]
Right, let's first figure out that problem in #110.

**Comment by UnrealKaraulov** [comment]
> I'll try to review the PR and get it in.

https://github.com/VerySleepy/verysleepy/pull/112
